wsgi.py

```diff
@@ -0,0 +1,32 @@
+import dash
+import flask
+from tesla_ver.bubble_chart import generateBubbleChart
+
+# Creates the flask server
+server = flask.Flask(__name__)
+
+# Creates the dashboard and connects it to the flask server
+app = generateBubbleChart(server=server)
+
+
+@server.route("/")
+def index():
+    """Renders the landing page"""
+    return """
+<html>
```

Comment:
I think that raw html should probably be placed in a separate file and then read into a variable. If you think that tesla-ver will need to be passing around html it might help to create a function like ```def get_html_from_file(Path path): ```

Reply:
I just put the raw html in there as a really brief placeholder to test the functionality.  I'd rather put html changes into a different PR, if at all possible.

Reply:
This is okay so long as you add a #TODO for it and open an issue

Reply:
Personally, I would air on the side of creating something simple to load html from a file and then expand upon it in a later PR. If you'd prefer, you can leave a ```# TODO: ``` in the code and add it as an Issue in the Tesla-ver repo.